Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I will merge and see if there are any conflicts. I just am not sure if your commit should have shown all the other files everyone else changed.